flask/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: '3' 
+services: 
+  web: 
+    build: app 
+    ports: 
+      - '5000:5000'
```

Comment:
nit: add a new line at the end of the file

Reply:
Done